cypress/e2e/seededFlows/adminFlows/articles/privilegedReactions.spec.js

```diff
@@ -0,0 +1,181 @@
+      'contains',
+      `/admin/content_manager/articles/${articleId}?tab=flags`,
+    );
+  });
+
+  it('should update url on quality reactions tab click', () => {
+    cy.findByRole('link', { name: 'Quality reactions' }).click();
+    cy.url().should(
+      'contains',
+      `/admin/content_manager/articles/${articleId}?tab=quality_reactions`,
+    );
+  });
+
+  it('should not contain any flags', () => {
+    cy.findByText('Article has no flags.').should('exist');
+  });
+
+  it('should not contain quality reactions', () => {
+    cy.findByRole('link', { name: 'Quality reactions' }).click();
+    cy.findByText('Article has no quality reactions by trusted users.').should(
+      'exist',
+    );
+  });
+
+  it('should display the correct thumb up count', () => {
+    cy.get('.flex .crayons-card:nth-child(1) .fs-s').should('contain', '0');
+  });
+
+  it('should display the correct thumb down count', () => {
+    cy.get('.flex .crayons-card:nth-child(2) .fs-s').should('contain', '0');
+  });
+
+  it('should display the correct vomit count', () => {
+    cy.get('.flex .crayons-card:nth-child(3) .fs-s').should('contain', '0');
+  });
+
+  it('should display the correct score', () => {
+    cy.get('.flex .crayons-card:nth-child(4) .fs-s').should('contain', '0');
+  });
+});
+
+describe('Article flagged by a user', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.intercept('POST', '/admin/reactions/*', (req) => {
+      req.reply((res) => {
+        const { id } = req.body;
+        const response = {
+          statusCode: 200,
+          body: {
+            outcome: 'Success',
+          },
+        };
+
+        if (req.body.removeElement === true) {
+          cy.get(`#${id}`).then(($element) => {
+            $element.remove();
+          });
+          cy.get(`#js__reaction__div__hr__${id}`).then(($element) => {
+            $element.remove();
+          });
+        } else {
+          cy.reload();
+        }
+
+        res.send(response);
+      });
+    });
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user).then(() => {
+        cy.visit('/admin/content_manager/articles/1'); // This article contains one flagged reaction.
+      });
+    });
+  });
+
+  it('should display the correct default data', () => {
+    cy.get('.flex .crayons-card:nth-child(3) .fs-s').should('contain', '1');
+  });
+
+  it('should display the correct user name', () => {
+    cy.get('.flex .crayons-subtitle-3').should(
+      'contain',
+      'Trusted User 1 \\:/',
+    );
+  });
+
+  it('should display the open flag status', () => {
+    cy.get('.flex .c-indicator').should('be.visible').as('flagStatus');
+    cy.get('@flagStatus')
+      .should('have.class', 'c-indicator--warning')
+      .and('contain', 'Open');
+  });
+
+  it('should display the correct score for open flag', () => {
+    cy.get('.flex .crayons-card:nth-child(2) .fs-s').should('contain', '-50');
+  });
```

Comment:
E2E is way slower than unit test, so it's more acceptable to test more than one aspect per block. In this case, these 4 specs can be merged into one and you can use comments to say what you are testing. ie
```
  it('should display the correct default data', () => {
    // show corrects user name
    ...

     // shows correct flag
     ...
  })
```

This way you check for all things in a single page load.

Reply:
Thanks for this suggestion. Based on this I have merged these tests to reduce them to almost half.